src/schema/sources.ts

```diff
@@ -126,6 +132,11 @@ export const typeDefs = /* GraphQL */ `
     Logged-in member object
     """
     currentMember: SourceMember
+
+    """
+    User squad permissions
+    """
+    permissions: [String]
```

Comment:
Why it's here? It should be in `SourceMember`

Reply:
This is so that when you load the Squad page, the member viewing the feed would already have all the information regarding whether they are able to do certain actions.

Or do you want to include it in the `SourceMember` then it would be depending on the client whether they want to include the property in the `currentMember`?